core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapCompareRequest.java

```diff
@@ -16,44 +16,48 @@
 package com.b2international.snowowl.core.request;
 
 import java.util.List;
-import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import javax.validation.constraints.Min;
 
 import com.b2international.snowowl.core.codesystem.CodeSystemRequests;
+import com.b2international.snowowl.core.compare.ConceptMapCompareConfigurationProperties;
 import com.b2international.snowowl.core.compare.ConceptMapCompareResult;
+import com.b2international.snowowl.core.compare.MapCompareSourceAndTargetEquivalence;
 import com.b2international.snowowl.core.domain.BranchContext;
 import com.b2international.snowowl.core.domain.ConceptMapMapping;
 import com.b2international.snowowl.core.domain.ConceptMapMappings;
 import com.b2international.snowowl.core.uri.ComponentURI;
+import com.google.common.base.Equivalence.Wrapper;
 import com.google.common.collect.ArrayListMultimap;
-import com.google.common.collect.HashMultimap;
 import com.google.common.collect.ListMultimap;
 import com.google.common.collect.Lists;
-import com.google.common.collect.Multimap;
 import com.google.common.collect.Sets;
 import com.google.common.collect.Sets.SetView;
 
 /**
 * @since 7.8
 */
 public final class ConceptMapCompareRequest extends ResourceRequest<BranchContext, ConceptMapCompareResult> {
+	public static final int DEFAULT_MEMBER_COMPARE_LIMIT = 50_000;
```

Comment:
Where do you use this? `5000` is set on the corresponding builder.

Reply:
It is used in the `CompareMappingSetsHandler`'s and `SnomedConceptMapCompareHandler`'s onFinish validation.

Reply:
Then move it to those classes, since that is UI specific and keep the API still load 5000 changes by default.